src/clients/SpokePoolClient.ts

```diff
@@ -453,6 +464,15 @@ export class SpokePoolClient extends BaseAbstractClient {
       }
     }
 
+    // Determine if this spoke pool has the capability to bridge UDSC via the CCTP token bridge.
+    // The CCTP bridge is canonically disabled if the `cctpTokenMessenger` is the ZERO address.
+    // Note: not all chains support CCTP so we should, hence the initial check
+    let hasCCTPBridgingEnabled = false;
+    if (chainIsCCTPEnabled(this.chainId)) {
+      const cctpBridgeAddress = String(await this.spokePool.cctpTokenMessenger());
```

Comment:
nit: should we make this call at the toBlock just to ensure the result is a bit more consistent with the other data in the update?

Reply:
Good callout - revised: https://github.com/across-protocol/sdk-v3/pull/631/commits/dd85be3c5fb124d99e035accd3da9543d1abb363